components/clp-package-utils/clp_package_utils/scripts/compress.py

```
@@ -63,6 +64,7 @@ def _generate_logs_list(
 
 def _generate_compress_cmd(
     parsed_args: argparse.Namespace,
+    dataset: Optional[str],
```

Flake8 (7.2.0)

[error] undefined name 'Optional'

(F821)